app/models/user.rb

```diff
@@ -1,9 +1,19 @@
 class User < ApplicationRecord
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: Relationship.name,
+    foreign_key: "follower_id", dependent: :destroy
+  has_many :passive_relationships, class_name: Relationship.name,
+    foreign_key: "followed_id", dependent: :destroy
+  has_many :following, through: :active_relationships, source: :followed
+  has_many :followers, through: :passive_relationships, source: :follower
```

Comment:
cu the cai doan khai bao nay di anh.
VD:  has_many :microposts, dependent: :destroy
user co nhieu micrpost, neu user bi xoa thi micropost bi xoa